CondCore/ESSources/plugins/CondDBESSource.cc

```diff
@@ -741,6 +738,45 @@ void CondDBESSource::fillTagCollectionFromDB(const std::vector<std::string>& con
   }
 }
 
+void CondDBESSource::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+
+  edm::ParameterSetDescription dbParams;
+  dbParams.addUntracked<std::string>("authenticationPath", "");
+  dbParams.addUntracked<int>("authenticationSystem", 0);
+  dbParams.addUntracked<std::string>("security", "");
+  dbParams.addUntracked<int>("messageLevel", 0);
+  dbParams.addUntracked<int>("connectionTimeout", 0);
+  desc.add("DBParameters", dbParams);
+
+  desc.add<std::string>("connect", std::string("frontier://FrontierProd/CMS_CONDITIONS"));
+  desc.add<std::string>("globaltag", "");
+  desc.add<std::string>("snapshotTime", "");
+  desc.addUntracked<std::string>("frontierKey", "");
+
+  edm::ParameterSetDescription toGetDesc;
+  toGetDesc.add<std::string>("record", "");
+  toGetDesc.add<std::string>("tag", "");
+  toGetDesc.add<std::string>("snapshotTime", "");
+  toGetDesc.add<std::string>("connect", "");
+  toGetDesc.add<unsigned long long>("refreshTime", std::numeric_limits<unsigned long long>::max());
+  toGetDesc.addUntracked<std::string>("label", "");
+
+  std::vector<edm::ParameterSet> default_toGet(1);
+  desc.addVPSet("toGet", toGetDesc, default_toGet);
```

Comment:
Why one empty as the default value?

Reply:
to validate the layout of the possible PSet, but not putting anything inside.